File: data/src/main/java/soup/movie/data/model/PlanMovieRequest.java
package soup.movie.data.model;

import java.util.HashMap;
import java.util.Map;

import soup.movie.data.base.QueryMapProvider;

public class PlanMovieRequest implements QueryMapProvider {

    public PlanMovieRequest() {
    }

    @Override
    public String toString() {
        return "PlanMovieRequest{}";
    }

    @Override
    public Map<String, String> toQueryMap() {
        return new HashMap<>();
    }
}

File: data/src/main/java/soup/movie/data/model/PlanMovieRequest.kt
package soup.movie.data.model

import java.util.HashMap

import soup.movie.data.base.QueryMapProvider

class PlanMovieRequest : QueryMapProvider {

    override fun toString(): String {
        return "PlanMovieRequest{}"
    }

    override fun toQueryMap(): Map<String, String> {
        return HashMap()
    }
}
